### PR TITLE
Move contribute info to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tonywu315**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ For API documentation, visit the [documentation page](https://capitalone.github.
 
 If you have suggestions or find a bug, [please open an issue](https://github.com/capitalone/dataprofiler/issues/new/choose).
 
+If you want to contribute, visit the [contributing page](https://github.com/capitalone/dataprofiler/.github/CONTRIBUTING.md).
```
Comment (JGSweets) [inline README.md:41]: not found

**Review by JGSweets — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (taylorfturner) [inline README.md:41]:
```suggestion
If you want to contribute, visit the [contributing page](https://github.com/capitalone/dataprofiler/blob/main/.github/CONTRIBUTING.md).
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -9,6 +9,32 @@ First off, thanks for your input! We love to hear feedback from the community an
...
+
+## Testing
+Before running unit tests, make sure you install the testing dependencies with `pip3 install -r requirements-test.txt`.
+
+To execute unit tests, run the following
+```cli
+DATAPROFILER_SEED=0 python3 -m unittest discover -p "test*.py"
+```
+
+For more nuanced testing runs, check out more detailed documentation [here](https://capitalone.github.io/DataProfiler/docs/0.7.11/html/install.html#testing).
```
Comment (taylorfturner) [inline .github/CONTRIBUTING.md:36]:
```suggestion
For more nuanced testing runs, check out more detailed documentation [here](https://capitalone.github.io/DataProfiler/docs/0.8.1/html/install.html#testing).
```

**Review by taylorfturner — APPROVED** [review]

**Review by taylorfturner — DISMISSED** [review]
URL fixes

**Review by JGSweets — APPROVED** [review]

**Review by micdavis — APPROVED** [review]

**Pull request merged**